test/JDBC/expected/drop_index-before-15_6-or-16_1-vu-cleanup.out

```diff
@@ -0,0 +1,11 @@
+use master
```

Comment:
Should cherry-pick this to BABEL4_X_DEV branch as well.

Reply:
drop_index is in test/JDBC/upgrade/latest/schedule; it was already merged before adding the upgrade tests